index.bs

```diff
@@ -1224,11 +1216,11 @@ run the following steps:
         |host| does not end with U+002D (`.`) followed by |domain|,
         then return failure.
     1. [=list/Append=] \``Domain`\`/|domain| ([=encoded=]) to |attributes|.
-1. If |secure| is false and |name| starts with either "`__Secure-`" or "`__Host-`",
+1. If |name| starts with either "`__Secure-`" or "`__Host-`",
```

Comment:
This seems like a behavior change - previously, this would fail with these prefixes only if _secure_ was false. Now it always fails with those prefixes. 

Was this previously wrong, or is this an intentional or accidental change?

Reply:
Ah sorry, you are right, this is an accidental change. 
But since |secure|  will be removed, should this line be removed?

Reply:
Removed this statement since it will never be non-secure